eng/versioning/version_client.txt

```diff
@@ -98,17 +98,17 @@ com.azure:azure-core-test;1.27.0-beta.8;1.27.0-beta.9
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.56;1.0.0-beta.57
 com.azure:azure-core-tracing-opentelemetry-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-version-tests;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos;4.68.0;4.69.0-beta.1
+com.azure:azure-cosmos;4.68.0;4.69.0
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos-encryption;2.19.0;2.20.0-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12;4.37.1;4.38.0-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12;4.37.1;4.38.0-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12;4.37.1;4.38.0-beta.1
+com.azure:azure-cosmos-encryption;2.19.0;2.20.0
+com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12;4.37.1;4.38.0
+com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12;4.37.1;4.38.0
+com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12;4.37.1;4.38.0
```

Comment:
Spark versions should be 4.37.2 (no new features - and azure-cosmos etc. is shaded)

Reply:
Fixed